src/lsp/superbol_free_lib/vscode_extension.ml

```diff
@@ -250,6 +250,20 @@ let contributes =
                         }
                 }
         } |})
+        ~initialConfigurations: [(
+          Manifest.any {| {
+                  "name": "SuperBOL: debug (launch)",
+                  "type": "superbol-gdb",
+                  "request": "launch",
+                  "preLaunchTask": "SuperBOL: build (debug)",
+                  "target": "${file}",
+                  "arguments": "",
+                  "cwd": "${workspaceFolder}",
+                  "group": [],
+                  "coverage": true,
+                  "verbose": false,
+                  "gdbtty": true
```

Comment:
To me, this is very annoying to have a terminal widow up on every debug session. If that's what people typically get on Windows then I'd be to find a way to enable that by default on that platform only.